WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderSiteHeaderView.java

```diff
@@ -158,7 +158,7 @@ private void showBlogInfo(ReaderBlog blogInfo, String source) {
         if (blogInfo.hasName()) {
             txtBlogName.setText(blogInfo.getName());
         } else {
-            txtBlogName.setText(R.string.reader_untitled_post);
+            txtBlogName.setText(UrlUtils.getHost(blogInfo.getUrl()));
```

Comment:
❓ Should we be checking here if `getUrl()` is not null/empty, and if it is, provide a fallback string (e.g. `"Untitled"` or `""`)? I suppose this should never happen, but I saw you're doing it in `SubfilterListItem:72` and thought it might be good to be on the safe side. WDYT?